presto-docs/src/main/sphinx/conf.py

```diff
@@ -114,6 +114,16 @@ def get_version():
     'version_json': '../versions.json',
 }
 
+base_url='https://prestosql.io'
+ogp_site_url='%s/docs/current/' % (base_url)
+ogp_site_name='%s %s Documentation' % (project, release)
+ogp_image='https://raw.githubusercontent.com/prestosql/presto/master/presto-docs/src/main/resources/logo/web/main/black/Presto_Logo_BlackBG-01.svg'
```

Comment:
We should link to an image in the documentation, so that it's on the same website. We can accomplish this by copying this image into the `presto-docs/src/main/sphinx/images` directory so that it gets published with the documentation.

Reply:
Yeah! I agree - I was not sure if and what image to use so I've used one that we have on github. I can copy and update link